felix/iptables/table.go

```diff
@@ -869,6 +869,7 @@ func (t *Table) attemptToGetHashesAndRulesFromDataplane() (hashes map[string][]s
 	waitErr := cmd.Wait()
 	if waitErr != nil {
 		log.WithError(waitErr).Warn("iptables save failed")
+		log.WithFields(log.Fields{"table": t.Name, "cmd": t.iptablesSaveCmd}).Warn("iptables save failed")
```

Comment:
```go
if log.IsLevelEnabled(log.DebugLevel) {
  t.logCxt.WithFields(log.Fields{"cmd": t.iptablesSaveCmd}).Warn("iptables save failed")
}
```

I think I suggested only doing this at debug level in a previous comment. It should look something like this ideally (since log.WithFields will otherwise be executed even if not running in debug).

We can also use `t.logCxt` which already includes the table and ipVersion fields.

Reply:
Got it. Sorry misunderstood the previous comment.
I just fixed it